docs/configuration.md

```diff
@@ -0,0 +1,199 @@
+
+There is also an optional key for all projects **additionalRegistrationPageText** which instead of containing a string should contain a list of strings. These strings will be displayed on the registration page below any user questions and above the ethics approval statement (if used) and the registration button. This section can be used to display any required information to the user, the initial motivation for this section was to provide the relevant licensing information for all of the images used on the website. It can also be useful to display a welcome message to users if no questions are being asked and the registration screen is just used as a login screen.
+
+### Comparison configuration
+
+This section needs to define all of the images you want to use organised by groups and for weighted item pairs all of the pairs and their associated weights. If you are not using weighted pairs this comparison configuration can be provided directly in the JSON or as a csv file. When using a csv file the csv file and the JSON configuration file should be put in the same directory and when running the setup and reset commands the path to the configuration file should instead be to the parent directory containing the two files.
+
+To see how the JSON should be configured please refer to the examples provided.
+
+Refer to `examples/config-equal-item-weights.json` or `examples/config-equal-item-weights-preference.json` to configure a scenario without any weighting for the item pairs and either with or without the item preference stage.
+
+Refer to `examples/config-custom-item-weights.json` to configure a scenario where custom weights will be defined for all item pairs.
+
+If the configuration is being provided in a csv file, then the JSON file must provide the name of the csv file as follows:
+
+```json
+"comparisonConfiguration" : {
+        "csvFile": "image-data.csv"
+    }
+```
+
+The csv file must contain a minimum of two columns and up to five columns. The column headers and their content are listed below. The order of the column is not important but the header names are: case is not relevant but the correct wording and spacing must be followed.
+
++ **image** - *required* - The file name of the image.
++ **item display name** - *required* - The string to be used to describe the image in the interface.
++ **item name** - *optional* - A string which identifies the image and only contains lowercase alpha numeric characters, underscored and dashes. This will be generated from item display name if the column is not provided but if there are any characters (with the exception of spaces) in the item display name column then validation errors will be raised.
```

Comment:
```suggestion
+ **item name** - *optional* - A string which identifies the image and only contains lowercase alpha numeric characters, underscores and dashes. This will be generated from the item display name if the column is not provided, but if there are any characters (with the exception of spaces) in the item display name column then validation errors will be raised.
```